src/main/java/homework/twnty/four/springmvcproducts/controller/ProductController.java

```diff
@@ -0,0 +1,47 @@
+package homework.twnty.four.springmvcproducts.controller;
+
+import homework.twnty.four.springmvcproducts.model.Product;
+import homework.twnty.four.springmvcproducts.repository.ProductRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+
+@Controller
+public class ProductController {
+
+    private ProductRepository productRepository;
+
+    @Autowired
+    public ProductController(ProductRepository productRepository) {
+        this.productRepository = productRepository;
+    }
+
+    @GetMapping
+    String getFromUser (Model model){
+        Product product = new Product();
+        model.addAttribute("product", product);
+        return "index";
+    }
+    @PostMapping("/")
```

Comment:
Jak nasłuchujesz na `/` to nie musisz podawać tego w adnotacji - wystarczy że napiszesz samo `@PostMapping` tak samo jak w przypadku `GetMapping`